packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -1,7 +1,16 @@
 <div data-ovh-alert></div>
 
-<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<div class="row">
+    <h1 data-translate="dedicated_server_interfaces_title" class="col-auto mr-auto"></h1>
+    <oui-guide-menu text="{{::'dedicated_server_interfaces_guides' | translate}}" class="col-auto px-5">
+        <oui-guide-menu-item data-href="https://docs.ovh.com/gb/en/dedicated/ola-manager/" external>
```

Comment:
url to be defined in localized app constants ? guides.url[subsidiary]

Reply:
@frenautvh please refer to my previous comment :)

https://github.com/ovh/manager/pull/1719#discussion_r342973181